### PR TITLE
Jkl 117 wire noti filetree rollback

### PR DESCRIPTION
**Pull request by BiPhan4**
Notifications module added and integrated with filetree. It still needs finalizing following discussion with Erin so it's "disabled" at the moment. Added:

- UUID as string
- makeRoot 
- removeViewers 
- general clean up

### CODE REVIEW

**Review by TheMarstonConnell — CHANGES_REQUESTED** [review]
Won't accept the PR until `ts-client` and `vue` are removed

File: x/filetree/keeper/access.go [hunk 1]
```diff
@@ -37,14 +37,16 @@ func HasEditAccess(file types.Files, user string) bool {
 	json.Unmarshal([]byte(peacc), &jvacc)
 
 	h := sha256.New()
-	h.Write([]byte(fmt.Sprintf("e%d%s", trackingNumber, user)))
+	h.Write([]byte(fmt.Sprintf("e%s%s", trackingNumber, user)))
 	hash := h.Sum(nil)
 
 	addressString := fmt.Sprintf("%x", hash)
+	fmt.Println("@@@@address string is", addressString)
```
Comment (TheMarstonConnell) [inline x/filetree/keeper/access.go:44]: I'd love if as a standardization we moved away from fmt and used the context's logger instead but will still approve with that

**Review by TheMarstonConnell — APPROVED** [review]
Thanks for fixing up my previous comment, looks good now!

File: x/filetree/keeper/access.go (hunk 1 above)
Comment (BiPhan4) [reply x/filetree/keeper/access.go:44]: My bad--I always remove those before making a PR but I missed this one. Learning how to use context's logger is definitely on the agenda. Many thanks.

**Pull request merged**